backends/advanced/src/advanced_omi_backend/routers/modules/system_routes.py

```
@@ -128,3 +128,41 @@
+async def update_memory_config_raw(
+    config_yaml: str,
+    current_user: User = Depends(current_superuser)
+):
+    """Update memory configuration YAML and hot reload. Admin only."""
+    return await system_controller.update_memory_config_raw(config_yaml)
+
+
+@router.post("/admin/memory/config/validate")
+async def validate_memory_config(
+    config_yaml: str,
+    current_user: User = Depends(current_superuser)
```

Ruff (0.12.2)

Unused function argument: `current_user`

(ARG001)